### PR TITLE
Fix LOC counting

### PR DESCRIPTION
**Pull request by decaz**
Latest coverage adds <sources> to the generated XML file.

### CODE REVIEW

File: clover/__init__.py [hunk 1]
```diff
@@ -75,15 +76,22 @@ def __init__(self, name, filename, statements, covered_statements,
...
-                for i, l in enumerate(f):
-                    pass
-        except IOError:
-            self.loc = 0
-        self.loc = i + 1
+        for source in sources:
+            filename = os.path.join(source, self.filename)
+            try:
+                with open(filename) as f:
+                    for i, __ in enumerate(f, start=1):
```
Comment (tumb1er) [inline clover/__init__.py:88]: may be `max(loc)` for each filename should be computed here? Now `i` is just number of lines in last opened file

Comment (tumb1er) [reply clover/__init__.py:88]: oops, only first existing file is opened (loop break below) - so, need some comments about what's going on here :-)

File: tests.py [hunk 2]
```diff
@@ -96,18 +92,15 @@ def testOpenCoverage(self):
...
+        statements = ncloc = 179
+        covered_conditions = 32
+        covered_statements = 151
+        conditions = 50
 
         if PY3:
             covered_conditions += 1
             covered_statements += 5
 
-        if not COV44:
```
Comment (tumb1er) [inline tests.py]: How will this pass with `coverage<4.4`?

**Review by tumb1er — CHANGES_REQUESTED** [review]
Build failing for coverage==3.7 and coverage==4.0 :-1:

File: clover/__init__.py (hunk 1 above)
Comment (decaz) [reply clover/__init__.py:88]: @tumb1er, fixed tests and added some comments.

**Pull request merged**